get-pmm.sh

```diff
@@ -73,7 +73,7 @@ setup_colors() {
 # Prints message to stderr with new line at the end.
 #######################################
 msg() {
-  echo >&2 -e "${1-}"
+  echo >&2 -ne "${1-}"
```

Comment:
what was the reason to get rid of new line at the end? I think this change increases risk to break formatting.

Reply:
I was originally playing with \r to reset cursor to the beginning of the line instead of scrolling but decided against it.  most lines already had newlines so I cleaned up the remaining ones (seems I missed one) but instead I think we'll let the native docker pull text show since it seems dumb to reinvent a spinning progress bar when the docker command already has it.